admin/app/controllers/admin/CommercialController.scala

```diff
@@ -74,13 +77,14 @@ class CommercialController(implicit context: ApplicationContext) extends Control
 
     val (hasNumericTestValue, hasStringTestValue) =
       lineItemsByAdTest partition { case (testValue, _) =>
-      def isNumber(s: String) = s forall Character.isDigit
+        def isNumber(s: String) = s forall Character.isDigit
```

Comment:
This will be true for an empty string.  I don't know if that matters.  But I see that this is just a reformatting anyway.